### ISSUE TITLE
Getting `Cannot find module 'typescript'` in CI environments using `npx auto`

### ISSUE THREAD

**Issue by reckter** [intuit/auto#1880]
**Describe the bug**
When running auto in an CLI (github actions in our example to be precise) environment using `npx`, it fails with `Cannot find module 'typescript'`

**To Reproduce**
Run `npx auto check-pr` (and probably other commands) in an CLI environment, without running `npm i` or related commands first

**Expected behavior**

Running out of the box, without requiring other dependencies to be installed.

**Screenshots**

Full stacktrace:
```
npx: installed 228 in 10.695s
Cannot find module 'typescript'
Require stack:
- /home/runner/.npm/_npx/1535/lib/node_modules/auto/node_modules/@auto-it/core/dist/auto.js
- /home/runner/.npm/_npx/1535/lib/node_modules/auto/dist/parse-args.js
- /home/runner/.npm/_npx/1535/lib/node_modules/auto/dist/bin/auto.js
```
running the command `npx auto check-pr`

**Environment information:**

Github actions CI.
using
- actions/setup-node@v2
- actions/checkout@v2

**Additional context**

Seems to be related to the commit https://github.com/intuit/auto/commit/be5c068ff336aa0a020f5b81a69943c3a4c274f8

**Comment by gflores-jahia** [comment]
Getting the same error when using yarn (v1.22.10):

**Comment by adierkens** [comment]
:rocket: Issue was released in [`v10.20.2`](https://github.com/intuit/auto/releases/tag/v10.20.2) :rocket:
